### PR TITLE
[CES-561] Create GitHub self hosted runner

### PR DESCRIPTION
**Pull request by kin0992**
Depends on #7

### CODE REVIEW

File: infra/github-runner/dev/locals.tf [hunk 1]
```diff
@@ -0,0 +1,12 @@
+locals {
+  prefix    = "dx"
+  env_short = "d"
+  repo_name = "dx-playground"
+
+  tags = {
+    CostCenter  = "TS310 - PAGAMENTI & SERVIZI"
```
Comment (gunzip) [inline infra/github-runner/dev/locals.tf:7]:
```suggestion
    CostCenter  = "TS700 - ENGINEERING"
```

Comment (kin0992) [reply infra/github-runner/dev/locals.tf:7]: @gunzip should we use that CostCenter for every resource?

Comment (Krusty93) [reply infra/github-runner/dev/locals.tf:7]: @gunzip I think CostCenter should be addressed to the right value almost everywhere (we're using the old TS310). Can you confirm TS is the right one?

Comment (gunzip) [reply infra/github-runner/dev/locals.tf:7]: yes, we should use TS700 for every dx resource

Comment (kin0992) [reply infra/github-runner/dev/locals.tf:7]: [This PR](https://github.com/pagopa/dx-playground/pull/21) is going to change the CostCenter accordingly.

**Comment by mamu0** [comment]
I think you can use the `github_runner` folder in this [PR](https://github.com/pagopa/dx/pull/189/files#diff-5da8a898bb28f98daba7e5a4c86bda202fff1626dd131147eaf7e4d65de34849), changing specific repository configuration

**Review by gunzip — APPROVED** [review]

**Pull request merged**